### DISCUSSION TITLE
Something broken from nui nvim

### DISCUSSION THREAD

**Discussion by directormac** [LazyVim/LazyVim#916] [Q&A]
![image](https://github.com/LazyVim/LazyVim/assets/5866196/d4076b06-c8de-478b-b709-795b00273a0a)

this happened as i updated like 40mins ago, the format on save now shows on top

when i try going command mode the first time it works, after awhile it does not anymore.

**Comment by folke** [comment]
related: https://github.com/folke/noice.nvim/issues/503

**Comment by folke** [comment]
Just pushed a work-around to noice to prevent the nui issue
https://github.com/folke/noice.nvim/commit/4db3c8f14302273b842f73a8facf1999169c1e41

Upstream issue https://github.com/MunifTanjim/nui.nvim/issues/270

**Comment by directormac** [comment]
As always thank you!
